modules/firefox/each-config.nix

```
@@ -0,0 +1,154 @@
+# manually applied args
+{
+  mkTarget,
+  name,
+  humanName,
```

```suggestion
  mkTarget,
  name ? "firefox",
  humanName ? "Firefox",
```

you probably want defaults set in case if the arguments are specified

---

This file is always imported by the main Firefox module, which always supplies these args. Why do we want defaults?

---

This is an internal function, there shouldn't be a case where those arguments are specified.